dashboard/test/controllers/api/v1/pd/application/principal_approval_applications_controller_test.rb

```diff
@@ -90,8 +87,6 @@ class PrincipalApprovalApplicationsControllerTest < ::ActionController::TestCase
         form_data: build(PRINCIPAL_APPROVAL_HASH_FACTORY,
           do_you_approve: "Other:",
           do_you_approve_other: "this is the other for do you approve",
-          committed_to_master_schedule: "Other:",
-          committed_to_master_schedule_other: "this is the other for master schedule",
           replace_course: "I don't know (Please Explain):",
           replace_course_other: "this is the other for replace course"
```

Comment:
Wasn't this question removed from the form?

Reply:
Yep -- removed. Also just removed the whole test above, `application update contains replaced courses`